src/language/language.ts

```diff
@@ -176,478 +182,301 @@ export const keywords = [
 ];
 
 export const builtinFunctions = [
-  'current_schema',
```

Comment:
Do we need to migrate old queries that are using these lowercase functions?

_(I have never touched this `language` file so not sure how it's used 😉 )_

Reply:
At the moment, the language is only used for providing syntax highlighting. Functions are purple in the code editor for example. Non tokenized functions and keywords can still be used - just not with syntax highlighting.

Reply:
So with this change, a query that used to be highlighted might no longer be (for the functions especially), but there will no change in terms of running/results of the query correct?
Sounds like a fair tradeoff for a better alignment 👍